docs/source/index.rst

```diff
@@ -30,6 +30,7 @@
     :caption: Documentation
 
     docs/cli
+    docs/log
```

Comment:
better to use logging to avoid confusion with "a log of the doc"....

Reply:
updated